index.js

```diff
@@ -34,34 +53,35 @@ module.exports = {
 
       var header = appConfig.contentSecurityPolicyHeader;
       var headerConfig = appConfig.contentSecurityPolicy;
-      var normalizedHost = options.host === '0.0.0.0' ? 'localhost' : options.host;
 
       if (options.liveReload) {
         ['localhost', '0.0.0.0'].forEach(function(host) {
-          headerConfig['connect-src'] = headerConfig['connect-src'] + ' ws://' + host + ':' + options.liveReloadPort;
-          headerConfig['script-src'] = headerConfig['script-src'] + ' ' + host + ':' + options.liveReloadPort;
+          var liveReloadHost = host + ':' + options.liveReloadPort;
+          headerConfig['connect-src'] += (' ' + 'ws://' + liveReloadHost);
+          headerConfig['script-src'] += (' ' + liveReloadHost);
```

Comment:
When wrapping `CSP_NONE` and `CSP_SELF` in arrays, these concatenations have to be replaced with `.push()`.